hack/deploy-local.sh

```diff
@@ -21,14 +21,33 @@ kubectl wait --for=condition=Ready pod -n spire-system --selector=app=spire-agen
 echo "next"
 
 cd safe || exit
+
 kubectl apply -f ./Namespace.yaml
 kubectl apply -f ./Role.yaml
+
+echo "should have created clusterrole and clusterrolebinding:"
+# Get clusterroles and clusterrolebindings
+clusterrole_output=$(kubectl get clusterrole)
+clusterrolebinding_output=$(kubectl get clusterrolebinding)
+# Check for the presence of "vsecm" in the clusterrole output
+echo "$clusterrole_output" | grep "secret-readwriter" > /dev/null
+clusterrole_status=$?
+# Check for the presence of "vsecm" in the clusterrolebinding output
+echo "$clusterrolebinding_output" | grep "secret-readwriter-binding" > /dev/null
+clusterrolebinding_status=$?
+# Exit if either clusterrole or clusterrolebinding is not present
+if [ $clusterrole_status -ne 0 ] || [ $clusterrolebinding_status -ne 0 ]; then
+    echo "Error: Required clusterrole or clusterrolebinding with 'vsecm' not found."
+    exit 1
+fi
```

Comment:
The script now errs out if the cluster role and role binding cannot be created for any reason.